Sources/Plasma/Apps/plClient/Mac-Cocoa/main.mm

```diff
@@ -209,7 +201,7 @@ - (id)init
 - (void)startRunLoop
 {
     [[NSRunLoop currentRunLoop] addPort:[NSMachPort port] forMode:@"PlasmaEventMode"];
-    [self.plsView setBoundsSize:self.plsView.bounds.size];
```

Comment:
Odd. But I think maybe this line was here to get the view to just enter setBoundsSize and do setup.